packages/graphiql/vite.config.ts

```diff
@@ -12,11 +12,11 @@ export default defineConfig({
   server: {
     port: 4001,
     proxy: {
-      '/graphql': 'http://localhost:4000',
+      '/graphql': 'http://localhost:8080/graphql',
     },
   },
   define: {
-    'process.env.NODE_ENV': '"production"',
+    // 'process.env.NODE_ENV': '"production"',
```

Comment:
this was necessary to get the vite server to actually function locally, however I'm not sure whether or not it should be removed altogether. Thoughts?